tests/v2/validation/charts/backup_restore/README.md

```diff
@@ -0,0 +1,41 @@
+# Backup Restore Operator (BRO)
+Backup Restore Operator (BRO for short) is a disaster recovery chart that can be installed on the local cluster only, and is used to create backups of Rancher resources contained in the `rancher-resource-set`. The resource set becomes available after the chart is installed and can be edited as a user needs. Once a backup is created and stored in either a local volume or in an S3 storage location (such as AWS S3 or Minio S3), a restore operation can be created to restore Rancher back to the backed up version of the Rancher resources.
+
+## Pre-requisites
+- Downstream RKE1 and RKE2 clusters are provisioned during the test to ensure the clusters come back into the active status after a restore.
+- Provisioning config is required because of this, example provisioningInput included below.
+- All tests require configs pulled in from the backupRestoreInput parameter.
+
+## Test Setup
+In your config file, set the following:
+```
+rancher: 
+  host: "rancher_server_address"
+  adminToken: "rancher_admin_token"
+  insecure: true/false
+  cleanup: true/false
+  clusterName: ""
+backupRestoreInput:
+  backupName: ""
+  s3BucketName: ""
+  s3FolderName: ""
+  s3Region: ""
+  s3Endpoint: ""
+  volumeName: ""
+  credentialSecretNamespace: ""
+  prune: true/false
+  resourceSetName: ""
+  accessKey: ""
+  secretKey: ""
+  rke1ClusterName: ""
+  rke2ClusterName: ""
+provisioningInput:
+  rke1KubernetesVersion:
+    - "v1.30.2-rancher1-1"
+  rke2KubernetesVersion:
+    - "v1.30.2+rke2r1"
+  k3sKubernetesVersion:
+    - "v1.30.2+k3s1"
```

Comment:
please keep the example versions generic so we don't have to update them again in the future